### DISCUSSION TITLE
How to make navigate sidebar and toc section collapsible?

### DISCUSSION THREAD

**Discussion by Soyoon-Choi** [squidfunk/mkdocs-material#7560] [Q&A]
I've read https://github.com/squidfunk/mkdocs-material/issues/3428 , but it seems like it is stopped to develope that feature. How do I customize the navigation sidebar and the table of contents section collapsible? I think it's possible by modifying the html file in partials, but I'd like to know how to do it specifically.

**Comment by squidfunk** [comment]
We are still considering this change, but we are currently working on [bigger things](https://squidfunk.github.io/mkdocs-material/blog/2024/08/19/how-were-transforming-material-for-mkdocs/) in the background. It will eventually be implemented, but we'll need some more time, as we're a very small team. Maybe another community member can share a solution that you can use right now before we ship native support ☺️
